dev_docs/older/release_model.html

```diff
@@ -50,7 +50,7 @@
 <p>CVS will not be released as a package and placed on the main
 download site. Users can obtain a copy of the CVS anonymously through
 sourceforge. Instructions for anonymous CVS access for non-developers
-can be found <a href="http://sourceforge.net/cvs/?group_id=44492">here</a>.</p>
+can be found <a href="https://sourceforge.net/projects/triplea/">here</a>.</p>
```

Comment:
As you said, the legacy docs need to be updated (or just removed).  The TripleA CVS repo doesn't even exist anymore. 😄